css/styles.css

```diff
@@ -599,6 +595,12 @@ div.about {
     background-image: url("../imgs/about-mebg-image_desktop.png");
     background-repeat: no-repeat;
     background-position-x: right;
+
+    /* position: absolute;
+    height: 354px;
+    width: 586px;
+    left: 37%;
+    top: 5%; */
```

Comment:
- [ ] Kindly remove all unused CSS codes from your codebase. It's good for your project to have a clean, professional, and lean codebase. Having commented on code makes it dirty. Please remove them 👍